runtime/compiler/env/VMJ9.cpp

```diff
@@ -5513,7 +5521,12 @@ TR_J9VMBase::getStringUTF8(uintptr_t objectPointer, char *buffer, intptr_t buffe
 uint32_t
 TR_J9VMBase::getVarHandleHandleTableOffset(TR::Compilation * comp)
    {
+#if defined(J9VM_OPT_METHOD_HANDLE) && (JAVA_SPEC_VERSION >= 11)
```

Comment:
Can the function be removed entirely and the callers ifdeffed?

Reply:
No. This code is still used in Java 11 to support VarHandles. We can't remove it until OJDK MHs are enabled/supported in Java 11.